### PR TITLE
Remove qemu_arm_launcher test from CI

### PR DESCRIPTION
**Pull request by tokatoka**

### CODE REVIEW

File: fuzzers/baby_fuzzer_minimizing/src/main.rs [hunk 1]
```diff
@@ -35,7 +35,7 @@ pub fn main() -> Result<(), Error> {
     let observer =
         unsafe { StdMapObserver::from_mut_ptr("signals", SIGNALS.as_mut_ptr(), SIGNALS.len()) };
 
-    let factory = MapEqualityFactory::with_observer(&observer);
+    let factory = MapEqualityFactory::new_from_observer(&observer);
```
Comment (domenukk) [inline fuzzers/baby_fuzzer_minimizing/src/main.rs:38]: ?

**Comment by domenukk** [comment]
Wouldn't it be better if @andreafioraldi fixed it? :P

**Comment by andreafioraldi** [comment]
> Wouldn't it be better if @andreafioraldi fixed it? :P

Nothing to fix, the makefile is just random garbage, it's not related to this fuzzer (that should not run in the CI IMO as bootstrapping a VM is a bit too much)

File: fuzzers/baby_fuzzer_minimizing/src/main.rs (hunk 1 above)
Comment (tokatoka) [reply fuzzers/baby_fuzzer_minimizing/src/main.rs:38]: api changed. with_observer doesn't exist

**Comment by tokatoka** [comment]
it turns out that I can't run qemu-based on the ci runner

**Pull request merged**